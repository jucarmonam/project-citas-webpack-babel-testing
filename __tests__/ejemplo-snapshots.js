//Snapshot son datos que se almacenan es un string y nos pueden servir para testear por ejemplo un objeto completo
const cliente = {
  nombre: "Juan 2",
  balance: 500,
  tipo: "Premium",
};

describe("Testin al cliente", () => {
  test("Es Juan Pablo", () => {
    expect(cliente).toMatchSnapshot();
  });
});
