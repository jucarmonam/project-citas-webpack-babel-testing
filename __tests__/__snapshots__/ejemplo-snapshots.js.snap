// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testin al cliente Es Juan Pablo 1`] = `
Object {
  "balance": 500,
  "nombre": "Juan 2",
  "tipo": "Premium",
}
`;
